modules/linear_boltzmann_solvers/discrete_ordinates_adjoint_solver/lbs_adj_solver.cc

```diff
@@ -76,8 +76,7 @@ DiscreteOrdinatesAdjointSolver::Initialize()
 void
 DiscreteOrdinatesAdjointSolver::Execute()
 {
-  primary_ags_solver_->Setup();
-  primary_ags_solver_->Solve();
+  ags_solver_->Solve();
```

Comment:
This applies to multiple places, but why is `ags_solver_->Setup();` not needed?

Reply:
`Setup` is really specific to the PETSc solvers, doing a lot of PETSc things that aren't needed for this solver... at least if we're using Classic Richardson.